browser/brave_shields/brave_shields_tab_helper.cc

```diff
@@ -384,8 +384,8 @@ bool BraveShieldsTabHelper::GetNoScriptEnabled() {
 }
 
 bool BraveShieldsTabHelper::GetForgetFirstPartyStorageEnabled() {
```

Comment:
this function only seems to be used from one place, where we know the current site URL - maybe worth removing?

Reply:
I'm not following, this is how ShieldsPanelDataHandler gets this for everything else, why would we change just this one as part of this PR? I agree that ShieldsPanelDataHandler/BraveShieldsTabHelper seem to have some redundancy, but the point of this PR is to deal with brave_shields_util